Installer/Product.wxs

```diff
@@ -1297,8 +1297,8 @@
       <Component Id="cmpB3EF7A92492F44029D4B8C1E55123D8D" Directory="dir643614AA5DD514F0CDE7FCAD4C22D6D2" Guid="a8b0ef18-d142-487b-91fd-d7e3cda088e0">
         <File Id="filA66AB3CD20094F17811461E2914765DC" KeyPath="yes" Source="$(var.SolutionDir)LabSources\PostInstallationActivities\SetupDscPullServer\SetupDscPullServerSql.ps1" />
       </Component>
-      <Component Id="cmpDE17B245EA354D77BC40DB5B0F43CDDA" Directory="dir643614AA5DD514F0CDE7FCAD4C22D6D2" Guid="e38f28c3-3de3-42a2-81ff-afe83e6f85aa">
-        <File Id="fil1492CDCF9CBB34982996FAA477019FB10" KeyPath="yes" Source="$(var.SolutionDir)LabSources\PostInstallationActivities\SetupDscPullServer\SetupSqlServerReportingServices.ps1" />
+      <Component Id="cmpDE17B245EA354D77BC40DB5B0F43CDDA" Directory="dir00729354D694424EA958809C47C9B720" Guid="e38f28c3-3de3-42a2-81ff-afe83e6f85aa">
+        <File Id="fil1492CDCF9CBB34982996FAA477019FB10" KeyPath="yes" Source="$(var.SolutionDir)LabSources\PostInstallationActivities\SqlServer\SetupSqlServerReportingServices.ps1" />
```

Comment:
Should we include this in the automation, then? Because I will 1000% forget about the installer again in case I add a post install activity... OK for now, but I would probably address this afterwards

Reply:
Yeah, I know. The whole installer concept should be changed one day.

Reply:
Found a lot of other issues that should be solved now. The config file works as well.